### PR TITLE
Ewok race haks

### PR DESCRIPTION
**Pull request by BamgaLab**
implement graphical assets (4 head models for each gender, 8 portraits for each gender, custom hand and body models.)

### CODE REVIEW

File: swlor2_parts/pfd0_bicepr208.mdl [hunk 1]
```diff
@@ -0,0 +1,499 @@
+# NeverBlender 4.0.5|Blender 4.0.2|2024-06-23
```
Comment (zunath) [inline swlor2_parts/pfd0_bicepr208.mdl:1]: All mdl files need to be compiled.

File: swlor2_2da/portraits.2da [hunk 2]
```diff
@@ -4455,3 +4455,19 @@
 4451   nautF10_       1      168    ****            0      ****          
 4452   nautF11_       1      168    ****            0      ****          
 4453   nautF12_       1      168    ****            0      ****          
+4454   ewokM1_        0      169    ****            0      ****
```
Comment (zunath) [inline swlor2_2da/portraits.2da:4458]: There are a bunch of portraits listed here but I don't see the equivalent tga files in the swlor2_portraits folder.

**Review by zunath — CHANGES_REQUESTED** [review]
Please review requested changes above.

**Comment by taelon875** [comment]
Double Check your TLK, name of race and race description are not showing in game.

**Comment by taelon875** [comment]
Female Ewok head 111 has face and covering as the same "hair" option.  This differs from all other heads.

**Pull request merged**